### DISCUSSION TITLE
How to view realtime querylog stream?

### DISCUSSION THREAD

**Discussion by r1bnc** [AdguardTeam/AdGuardHome#3481] [Q&A]
I am using Docker version of AdGuardHome the stable version 0.106.3, how do I view the real time query log stream similar to `pihole -t`? Trying to monitor an app that still shows ads even though I have set multiple blocklist.
An alternative solution that I think of, is to set the `query_log_mem_size` to maybe 10, then just `tail -f querylog.json` but, the data inside it is huge. I just want something like what pihole displays, domain, ip..

Thank you in advance

PS. Does anyone has made a good parser for the querylog json file?

**Comment by ameshkov** [comment]
There's no way to do that right now, but I think there's already a feature request about that.

**Reply by r1bnc** [reply]
Thank you, I've found something using jq and tail. Seems enough in my case

**Comment by r1bnc** [comment]
Possible solution
https://github.com/AdguardTeam/AdGuardHome/issues/1428
https://github.com/AdguardTeam/AdGuardHome/issues/1821#issuecomment-647551928
